tabcmd/commands/auth/session.py

```diff
@@ -152,7 +153,7 @@ def _set_connection_options(self) -> TSC.Server:
         # args still to be handled here:
         # proxy, --no-proxy,
         # cert
-        http_options = {}
+        http_options = {"headers": {"User-Agent": "Tabcmd" + version}}
```

Comment:
Let's use the standard `name/version` format i.e. https://github.com/tableau/server-client-python/blob/master/tableauserverclient/server/endpoint/endpoint.py#L45